bps/bps_ApTemplate.yaml

```diff
@@ -10,38 +10,31 @@ campaign: DM-xxxxx
 
 submitPath: ${PWD}/bps/{outCollection}
 computeSite: ncsapool
-# Memory allocated for each quantum, in MBs
+# The default BPS walltime is on the order of 3 days.
+# That's much more than the vast majority of jobs need,
+# and it can cause your jobs not to run if you're too close to a maintenance window.
+# Set the default walltime as appropriate for what you are running (12 hours below).
+site:
+  ncsapool:
+    profile:
+      condor:
+        +Walltime: 43200
+# Default memory allocated for each quantum, in MBs; can be overridden on a per-task basis.
```

Comment:
This comment-doc line is different in the ApTemplate and ApPipe files, please choose one and have it for both.